CHANGES.rst

```diff
@@ -30,4 +30,12 @@ New Features
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Bundled copy of astropy-helpers upgraded to v1.0. [#251]
+- Update astropy-helpers to v1.0. [#251]
+- Update astropy-helpers to v1.0.1 [#258]
+- Update astropy-helpers to v1.0.2 [#260]
```

Comment:
I guess we only need to keep the last update in the changelog only? The others only affected the dev branch, not the release.